src/server/server.js

```diff
@@ -29,11 +29,17 @@ class Server {
         try {
             this.server = await this._listenPromisify();
 
-            log(`Server started on ${this.host}:${this.port}`);
+            log(`Server '${this.host}:${this.port}' started`);
         }
         catch (err) {
-            error(`Server failed to start`);
-
+            switch (err.code) {
+                case 'EACCES':
```

Comment:
Looks like we don't re-throw an error in these cases, do not log the original error and don't log all other errors at all.